README.md

```diff
@@ -1,62 +1,82 @@
 ---
 ## tl;dr
-**Complete as many as you can in 1 hour in any order you like. Ask questions!**
+**Complete as many as you can in 1 hour in any order you like. Ask questions (seriously)!**
 
 ---
-# Tasks:
+# Tasks
 
+## Part 1
+_Goal: Connect to a virtual machine using `ssh` and configure git_
 - Login to EC2 instance
   - SSH into EC2 instance using your assigned username and SSH key pair
-- Clone a git repo onto the EC2 Instance
+- Clone this git repo onto the EC2 Instance
   - Make sure that your `apt-get` package lists are updated
-  - Install Git using the builtin package management tool
-  - Using `git config` Configure Git with your Github user name and email address
-  - Clone this Git repository to your home directory
+  - Install git using the builtin package management tool
+  - Use `git config` to configure git with your github user name and email address
+  - Clone this git repository to your home directory
 - Set up workspace
-  - Inside the local directory created by the cloned Git repository
+  - Inside the local directory created by the cloned git repository
     - Use `git branch` to create a local branch with the following name format `<first initial>-<last name>/tech-challenge`
+
+## Part 2 
+_Goal: Containerize an existing app using Docker_ 
 - Install Docker 
   - Install the stable release of `docker-ce` and its pre-requisites for the correct Linux distribution
 - Build a Docker image 
-  - Using `docker build`, build an image from the Dockerfile within the cloned Git repository
+  - Using `docker build`, build an image from the Dockerfile within the `tethys` repository
 - Run and configure image
   - Read the documentation within the cloned repository and familiarize yourself with the application within
   - Understand the configuration options necessary for the Docker image you just built to run successfully
-  - Run the Docker image you just built using `docker run` providing the environment variables necessary and a  `command` option
-  - Ensure the web service is exposed and available on port 80 of the EC2 instance
+    - Add a `CMD` directive to your Dockerfile to make sure the python app runs within the container
+  - Run the Docker image you just built using `docker run`, and include the following options: 
+    - Set the `PORT` environment variable to `8080`
+    - Publish the container to port 4000 and map it to 8080 
 - Modify `Dockerfile`
   - Using the Dockerfile `COMMAND` or `ENTRYPOINT` directives, ensure that the docker image built can be run without the `command` flag being specified 
   - Test your Dockerfile changes by re-building the Docker image and testing it by re-running
-  - Once you’re satisfied, use Git to commit your changes to the Dockerfile to your local Git branch
-- Make Docker web service survive an EC2 instance restart
+  - Once you’re satisfied, use git to commit your changes to the Dockerfile to your local git branch
+
+- (Optional) Make Docker web service survive an EC2 instance restart
```

Comment:
I think I prefer not say _optional_ here... I think. 
Theoretically, all the tasks are 'optional'  and the candidate should choose where their strengths are so, perhaps better language around that should be added instead? Is that crazy?